revolv/static/css/screen.css

```diff
@@ -3700,6 +3702,181 @@ header.edit-project-header > .container > h1 {
   line-height: 24px;
 }
 
+/*mystyle start 29-june here*/
+.details-active-project-module .txt-area1 {
+  float: right;
+  margin-bottom: 45px;
+  margin-left: 0 !important;
+}
+.txt-area1 .module-box {
+  background: #ededed none repeat scroll 0 0;
+  margin: 0 0 32px;
+  min-height: 200px;
+  padding: 24px;
+  text-align: center;
+}
+.txt-area1 .module-box.payment-module h5.title-h5 {
+  border-bottom: 1px solid #c3c3c3;
+  border-top: 1px solid #c3c3c3;
+  color: #4b4b4b;
+  font-family: "Source Sans Pro";
+  font-size: 16px;
+  font-weight: 700;
+  height: 33px;
+  line-height: 33px;
+  margin: 10px 0;
+}
+.txt-area1 .module-box .back-the-project {
+  color: #7d7d7d;
+  font-size: 14px;
+  line-height: 25px;
+  margin: 15px 0;
+}
+.txt-area1 .module-box .inputs label {
+  color: #4c4c4c;
+  font-size: 16px;
+  font-weight: bold;
+  margin: 5px 5px 10px;
+}
+.txt-area1 .module-box .inputs {
+  text-align: left;
+}
+.txt-area1 .btn-i-want-to-donate {
+  background-color: #00a1d9;
+  font-size: 14px !important;
+  height: 45px;
+  letter-spacing: 4px;
+  line-height: 45px;
+  margin-top: 2.5rem;
+  padding-left: 20px;
+  padding-top: 0;
+  width: 100%;
+}
+.details-active-project-module .container {
+  padding: 4px 54px 0 78px;
+}
+/*$ 20 CSS START HERE*/
```

Comment:
Same thing with this comment. Please update it to describe what the styles below apply to.